src/styles.css

```diff
@@ -2,64 +2,56 @@
 	font-family: 'Red Hat Text';
 	font-style: normal;
 	font-weight: 400;
-	src: url(chrome-extension://__MSG_@@extension_id__/fonts/RedHatText-Regular.ttf),
-		url(moz-extension://__MSG_@@extension_id__/fonts/RedHatText-Regular.ttf);
+	src: local(RedHatText-Regular.ttf);
```

Comment:
As I understand it, `local()` loads the font if it is already installed. But can we assume that everyone who uses the extension has this font installed? Supplying it only as a local file is not enough for this, as I understand it. Or am I wrong?